File: Practica 4/Ej6/factorial.c
#include <stdio.h>

int factorial(int num)
{
    int total = num;
    for (int i = num - 1; i != 0; i--)
    {
        total *= i;
    }

    return total;
}

int main()
{
    int num;

    printf("Introduce un número: ");
    scanf("%d", &num);

    printf("Factorial de %d es: %d", num, factorial(num));
}

File: Practica 6/Ej6/factorial.py
def main():
    n = int(input("Introduce un número entero: "))
    print(f"Factorial de {n} ({n}!) es: {factorial(n)}")


def factorial(num):
    total = 1
    for n in range(num):
        total *= num - n

    return total

main()